nimbus/src/enrollment.rs

```diff
@@ -412,6 +412,7 @@ pub enum EnrollmentStatus {
         enrollment_id: Uuid, // Random ID used for telemetry events correlation.
         reason: EnrolledReason,
         branch: String,
+        feature_id: String
```

Comment:
nit: trailing `,` will make future diffs cleaner :)

Reply:
OK, I just ran cargo fmt, which took care of this.